mne/viz/ica.py

```diff
@@ -775,7 +775,8 @@ def _plot_sources_raw(ica, raw, picks, exclude, start, stop, show, title,
                   ch_start=0, t_start=start, info=info, duration=duration,
                   ica=ica, n_channels=n_channels, times=times, types=types,
                   n_times=raw.n_times, bad_color=bad_color, picks=picks,
-                  first_time=first_time, data_picks=[], decim=1)
+                  first_time=first_time, data_picks=[], decim=1,
+                  noise_cov=None, whitened_ch_names=())
```

Comment:
@larsoner why are we doing this actually? Just because we can or because people will need this. How does our whitening interact with ICA?

Reply:
Need to pass the params / API uniformity internally. They shouldn't use it with ICA which is why it's not exposed in the public facing APIs for ICA plotting.